src/index.html

```diff
@@ -17,6 +17,87 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="/"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo.png"
+        />
+      </a>
+      <nav>
+        <ul>
+          <li>
+            <a
+              href="#"
+              class="nav-link is-active apple"
+            >
+              APPLE
+            </a>
+            <div class="rectangle"></div>
+            <!-- Elemento retângulo abaixo do link -->
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-link samsung"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-link smartphones"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-link colorido laptopsecomputers"
+              data-qa="is-active"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-link gadgets"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-link tablets"
+            >
+              TABLETS
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-link photo"
+            >
+              PHOTO
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-link video"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
```

Comment:
Add empty lines between multiline sibling blocks of HTML to improve readability. For example, add an empty line before and after the `<nav>` block.